frontend/assets/css/prime_menubar.scss

```diff
@@ -26,6 +26,14 @@
         padding: var(--padding-small) var(--padding-large);
         border-radius: var(--border-radius);
         height: 28px;
+        .text {
+          // this hack is necessary to make the clickable area as big as the parent
+          &::before{
+            content: "";
+            position: absolute;
+            inset: 0;
+          }
+        }
```

Comment:
**Suggestion:** You might want to do the same for this scenario:
![image](https://github.com/gobitfly/beaconchain/assets/114066135/a5436e71-3778-47da-bfef-d82b106b2f44)
i.e. when the drop-down menu is shown. The grey hover area OUTSIDE of the red rectangle shows a pointer but is not clickable.

For this scenario to happen, you have to be logged in and you have to have at least 2 validator dashboards.